usecases/src/main/java/com/architectcoders/usecases/IUseCase.kt

```diff
@@ -0,0 +1,8 @@
+package com.architectcoders.usecases
+
+import com.architectcoders.data.repository.CharacterRepository
+import com.architectcoders.domain.characters.Characters
+
+interface IUseCase <T,K>{
+ suspend fun invoke(param: K): T?
```

Comment:
@jujoru that's awesome! shouldn't be better if we respected the order of parameters. I mean, T as a function parameter and K as a returned object

Reply:
Perfect, I change this.